src/pages/admin/profile/[user_id].tsx

```diff
@@ -0,0 +1,202 @@
+  }
+  if (!targetUserData) return <Loading />;
+  return (
+    <>
+      <Suspense fallback={<Loading />}>
+        <Navbar />
+        <Stack
+          spacing={2}
+          alignItems="center"
+          justifyContent="center"
+          style={{ minHeight: "90vh" }}
+        >
+          <Typography variant="h1" align="center" sx={profile_layout}>
+            {targetUserData.username}
+          </Typography>
+          <Typography variant="body2" align="center" sx={profile_layout}>
+            {targetUserData.email}
+          </Typography>
+          <Avatar sx={avatar} alt="Profile picture" src={targetUserData.image as string} />
+
+          {targetUserData.isVerified &&
+            <VerifyChip />
+          }
+          <Stack direction="row" spacing={1}>
+            <Chip
+              icon={
+                targetUserData.sex === Gender.male ? (
+                  <MaleIcon />
+                ) : targetUserData.sex === Gender.female ? (
+                  <FemaleIcon />
+                ) : targetUserData.sex === Gender.others ? (
+                  <TransgenderIcon />
+                ) : (
+                  <div></div>
+                )
+              }
+              label={targetUserData.sex}
+            />
+            <Chip icon={<CakeIcon />} label={targetUserData.birthdate} />
+          </Stack>
+
+          {targetUserData.description.split("\n").map((row) => (
+            <Typography
+              variant="body1"
+              sx={{ ...profile_layout, wordBreak: "break-word" }}
+              key={row}
+            >
+              {row}
+            </Typography>
+          ))}
+          {!targetUserData.isVerified &&
+            <Button style={{ marginTop: "50px" }} onClick={openVerifyModal} variant="contained">
+              Verify
+            </Button>
+          }
+          <AdminVerifyDialog
+            openModal={isVerifyModalShow}
+            handleCloseModal={closeVerifyModal}
+            verifyUser={verifyUser}
+            nationalID={nationalIDCard}
+          >
+            <Box sx={{ width: "70%", margin: "auto" }}>
+              <TextField
+                placeholder="เลขโดด 13 หลักเท่านั้น"
+                error={isError || nationalIDCard.length > CHAR_LIMIT.MAX_NATIONAL_ID_CARD_NUMBER}
+                value={nationalIDCard}
+                onChange={handleNationalIDCardChange}
+                fullWidth
+                inputProps={{
+                  sx: {
+                    textAlign: "center",
+                    "&::placeholder": {
+                      textAlign: "center",
+                    },
+                  },
+                }}
+              />
+              <Box sx={helperText}>
+                <FormHelperText>
+                  {`${nationalIDCard.length}/${CHAR_LIMIT.MAX_NATIONAL_ID_CARD_NUMBER}`}
+                </FormHelperText>
+                {isError && <FormHelperText error>{errMsg}</FormHelperText>}
+              </Box>
+            </Box>
```

Comment:
Why you have to sent child props into adminVerifyDialog not implement in inside adminVerifyDialog component?

Reply:
I have already refactored this.